modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx

```diff
@@ -245,7 +245,7 @@ export default function ObjectFieldFormBase({
 			oneToManyRelationship &&
 			oneToManyRelationship.deletionType !== 'disassociate'
 		) {
-			return false;
+			return true;
```

Comment:
Hi @larissacribeiro, this will change the behavior for other cases that should not be impacted. The case described in the ticket should already be covered by this [condition](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/ObjectFieldFormBase.tsx#L241). You need to investigate why it's not working.

Reply:
Hi @naraandrade, thanks for reviewing. The issue was that the component was not receiving the objectDefinition, and that's why it never checked the condition. I've just updated it the fix!